<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pump manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="pumptable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPumps">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionManage"/>
    <addaction name="actionLocalisation"/>
    <addaction name="actionFrequency"/>
    <addaction name="actionType"/>
    <addaction name="actionStructure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPumps"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="text">
    <string>Pump</string>
   </property>
  </action>
  <action name="actionLocalisation">
   <property name="text">
    <string>Localisation</string>
   </property>
  </action>
  <action name="actionFrequency">
   <property name="text">
    <string>Frequency</string>
   </property>
  </action>
  <action name="actionType">
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionStructure">
   <property name="text">
    <string>Structure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
